chain-impl-mockchain/tests/common/address.rs

```diff
@@ -1,14 +1,30 @@
 use chain_addr::{Address, Discrimination, Kind, KindType};
-use chain_impl_mockchain::account::SpendingCounter;
-use chain_impl_mockchain::key::{EitherEd25519SecretKey, SpendingPublicKey};
+use chain_impl_mockchain::{
+    account::SpendingCounter,
+    key::{EitherEd25519SecretKey, SpendingPublicKey},
+    transaction::{Input, Output, UtxoPointer},
+    value::Value,
+};
+use std::fmt::{Debug, Formatter, Result};
```

Comment:
Please don't import `fmt::Result`, it makes the code confusing.
It's more common to import `std::fmt::self` and qualify with `fmt::Result` in usage.

Reply:
I change it to `use std::fmt;` somehow `std::fmt::self;` throws error

Reply:
Sure; I meant in a grouped shorthand like `use std::fmt::{self, Debug};`.